tests/test_iso_rxtx_fastpacket/src/main.xc

```diff
@@ -1,192 +1,42 @@
-{
-    unsigned char buffer[1024];
-    unsigned char x;
-
-    for (int i = 0; i < length; i++)
-    {
-        buffer[i] = g_txDataCheck[epNum]++;
-
-        //asm("ld8u %0, %1[%2]":"=r"(x):"r"(g_txDataCheck),"r"(epNum));
-       // read_byte_via_xc_ptr_indexed(x, p_txDataCheck, epNum);
-
-        //buffer[i] = x;
-        //x++;
-        //asm("st8 %0, %1[%2]"::"r"(x),"r"(g_txDataCheck),"r"(epNum));
-        //write_byte_via_xc_ptr_indexed(p_txDataCheck,epNum,x);
-    }
-
-    XUD_SetBuffer(ep, buffer, length);
-}
-
-
-#pragma unsafe arrays
-int RxDataCheck(unsigned char b[], int l, int epNum)
-{
-    int fail = 0;
-    unsigned char x;
-
-    for (int i = 0; i < l; i++)
-    {
-        unsigned char y;
-        //read_byte_via_xc_ptr_indexed(y, p_rxDataCheck, epNum);
-        if(b[i] != g_rxDataCheck[epNum])
-        {
-            printstr("#### Mismatch on EP: ");
-            printint(epNum); 
-            printstr(". Got:");
-            printhex(b[i]);
-            printstr(" Expected:");
-            printhexln(g_rxDataCheck[epNum]);
-            //printintln(l); // Packet length
-            return 1;
-        }
-
-        g_rxDataCheck[epNum]++;
-        //read_byte_via_xc_ptr_indexed(x, p_rxDataCheck, epNum);
-        //x++;
-        //write_byte_via_xc_ptr_indexed(p_rxDataCheck,epNum,x);
-    }
-
-    return 0;
-}
-
-
-#pragma unsafe arrays
-int TestEp_Bulk_Tx(chanend c_in1, int epNum1)
-{
-    XUD_ep ep_in1  = XUD_InitEp(c_in1);
-    
-    unsigned char buffer[PKT_COUNT][1024];
-
-    int counter = 0;
-    int length = INITIAL_PKT_LENGTH;
-
-    for(int i = 0; i< PKT_COUNT; i++)
-    {
-        for(int j = 0; j < length; j++)
-        {
-            buffer[i][j] = counter++;
-        }
-        length++;
-    }
-
-    length = INITIAL_PKT_LENGTH;
-
-#pragma loop unroll
-    for(int i = 0; i < PKT_COUNT; i++)
-    {
-        XUD_SetBuffer(ep_in1, buffer[i], length++);
-    }
-
-    exit(0);
-
-}
-#pragma unsafe arrays
-int TestEp_Bulk_Rx(chanend c_out1, int epNum1)
-{
-    // TODO check rx lengths
-
-    unsigned int length[PKT_COUNT];
-    XUD_Result_t res;
-
-    XUD_ep ep_out1 = XUD_InitEp(c_out1);
-
-    /* Buffer for Setup data */
-    unsigned char buffer[PKT_COUNT][1024];
-
-    /* Receive a bunch of packets quickly, then check them */
-#pragma loop unroll
-    for(int i = 0; i < PKT_COUNT; i++)
-    {
-        XUD_GetBuffer(ep_out1, buffer[i], length[i]);
-    }
-#pragma loop unroll
-    for(int i = 0; i < PKT_COUNT; i++)
-    {
-        RxDataCheck(buffer[i], length[i], epNum1);       
-    }
-
-    while(1);
-
-}
+#ifndef PKT_LENGTH_END
+#define PKT_LENGTH_END	   (19)
+#endif
 
+/* Endpoint type tables */
+XUD_EpType epTypeTableOut[EP_COUNT_OUT] = {XUD_EPTYPE_CTL, XUD_EPTYPE_ISO, XUD_EPTYPE_ISO, XUD_EPTYPE_ISO, XUD_EPTYPE_ISO, XUD_EPTYPE_ISO};
+XUD_EpType epTypeTableIn[EP_COUNT_IN] =   {XUD_EPTYPE_CTL, XUD_EPTYPE_ISO, XUD_EPTYPE_ISO, XUD_EPTYPE_ISO, XUD_EPTYPE_ISO, XUD_EPTYPE_ISO};
 
```

Comment:
Quite a few main.xc files `#include "test_main.xc"` and use its common functions.  This file does not.  Is that intentional or an oversight?